src/store/store.js

```diff
@@ -0,0 +1,24 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  state: {
+    drawer: false,
+  },
+  getters: {
+  },
+  mutations: {
+    setDrawer(state, value) {
+      state.drawer = value;
+    },
+  },
+  actions: {
+    setDrawer({ commit }, value) {
+      commit('setDrawer', value);
+    },
+  },
+  modules: {
+  },
+});
```

Comment:
A two way binding (cf [v-model](https://v2.vuejs.org/v2/guide/components.html#Using-v-model-on-Components) or [.sync](https://v2.vuejs.org/v2/guide/components-custom-events.html#sync-Modifier)) will be less heavy than a whole store for just a menu that `App.vue` can handle + it's totally overkill...